src/components/AutoCompleteSuggestions/index.js

```diff
@@ -26,13 +36,26 @@ function AutoCompleteSuggestions(props) {
         return () => (container.onpointerdown = null);
     }, []);
 
-    return (
+    React.useEffect(() => {
+        if (!parentContainerRef || !parentContainerRef.current) {
+            return;
+        }
+        parentContainerRef.current.measureInWindow((x, y, w) => setContainerState({left: x, bottom: windowHeight - y, width: w}));
+    }, [parentContainerRef, windowHeight, windowWidth]);
+
+    const componentToRender = (
         <BaseAutoCompleteSuggestions
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...props}
             ref={containerRef}
         />
     );
+
+    if (!width) {
+        return componentToRender;
+    }
+
+    return ReactDOM.createPortal(<View style={StyleUtils.getBaseAutoCompleteSuggestionContainerStyle({left, width, bottom})}>{componentToRender}</View>, document.querySelector('body'));
```

Comment:
Not a _huge_ deal, but I think we should strive to use the context-based `@gorhom/portal` instead of `ReactDOM.createPortal` to reduce unnecessary differences between code across platforms.

If we can keep our code in pure React Native (w/ react-native-web of course), we should